frontend/src/components/regions/CardDetails/Review.js

```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { useState } from "react";
```

Comment:
I know this is a matter of preference, but across the entire code base,  for consistency, how about we maybe continue to use them as they are used in other files?

React.useState() on the line where the hook is being used

Reply:
global searched, not necessity for this

Reply:
Hey Viro, you are saying there is no necessity for the review?

Reply:
Yes, already approved and moved it accordingly @Sbonelo01